server/models/review-request.js

```diff
@@ -42,4 +42,34 @@ const reviewSchema = new mongoose.Schema({
   },
 });
 
+// request model
+const requestSchema = new mongoose.Schema({
+  // References user owner
+  userOwner: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+  },
+
+  selectedReviewer: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+  },
+
+  // Needed for timeout eventually
+  requestCreateDate: {
+    type: Date,
```

Comment:
You can use timestamps options for this.
[Options: Timestamps doc](https://mongoosejs.com/docs/guide.html#timestamps)

Reply:
That is very useful. Making changes here shortly.